test/sql/window/test_moving_quantile.test_slow

```diff
@@ -0,0 +1,41 @@
+# name: test/sql/window/test_moving_quantile.test_slow
+# description: Moving quantile performance
+# group: [window]
+
+require vector_size 512
+
+# Integer benchmark tables
+statement ok
+create table rank100 as
+    select b % 100 as a, b
+    from (
+        select * from range(0, 10000000) tbl(b)
+    ) q;
+
+#   Fixed window
+statement ok
+prepare fwm as
+select sum(m)
```

Comment:
Can we check the result here?

Reply:
There are 10M values, but I will add a checksum.